### PR TITLE
Initial work

### PR DESCRIPTION
**Pull request by ryan-talley**

### CODE REVIEW

**Comment by ryan-talley** [comment]
Note these commits will be collapsed upon PR approval so I will collapse and merge them myself.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pyinotify==0.9.6
+ipaddress==1.0.17
+PyJWT==1.4.0
```
Comment (edarzins) [inline requirements.txt]: Please take this opportunity to prune these dependencies. I know that PyJWT and sseclient are not used here, they were cut and pasted a long time ago from the marathon controller

Comment (edarzins) [reply requirements.txt]: Where is the dependency on CCCL?

Comment (ryan-talley) [reply requirements.txt]: Good catch on the unneeded dependencies I will prune them. The CCCL dependency is now in the setup.py file as a dependency_links that will get installed as an install_requires.

**Pull request merged**